lib/src/client/transport/xhr_transport.dart

```diff
@@ -81,13 +83,11 @@ class XhrTransportStream implements GrpcTransportStream {
       if (_incomingProcessor.isClosed) {
         return;
       }
-      // Use response over responseText as most browsers don't support
-      // using responseText during an onProgress event.
-      final responseString = _request.response as String;
+      final responseText = _request.responseText;
```

Comment:
@minoic do you know?

I will try to look into this otherwise when I have the chance.

Reply:
This looks safe.

https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/responseText

MDN reports it as:

> Widely available
This feature is well established and works across many devices and browser versions. It’s been available across browsers since July 2015.